spring-cloud-dataflow-docs/src/main/asciidoc/tasks.adoc

```diff
@@ -360,3 +360,212 @@ Another option to start a task using the `task-launcher` would be to create a st
+╠══════════════════════════╪═══╪════════════════════════════╪════════════════════════════╪═════════╣
+║my-composed-task-timestamp│713│Wed Apr 12 16:43:07 EDT 2017│Wed Apr 12 16:43:07 EDT 2017│0        ║
+║my-composed-task-mytaskapp│712│Wed Apr 12 16:42:57 EDT 2017│Wed Apr 12 16:42:57 EDT 2017│0        ║
+║my-composed-task          │711│Wed Apr 12 16:42:55 EDT 2017│Wed Apr 12 16:43:15 EDT 2017│0        ║
+╚══════════════════════════╧═══╧════════════════════════════╧════════════════════════════╧═════════╝
+```
+In the example above we see that my-compose-task launched and it also launched
+the other tasks in sequential order.
+
+==== Destroying a Composed Task
+The same command used to destroy a stand-alone task is the same as destroying a
+composed task.  The only difference is that destroying a composed task will
+also destroy the child tasks associated with it.   For example
+
+```
+dataflow:>task list
+╔══════════════════════════╤═══════════════════════════════════════════════════════════════
+║        Task Name         │                      Task Definition
+╠══════════════════════════╪═══════════════════════════════════════════════════════════════
+║my-composed-task          │mytaskapp && timestamp
+║my-composed-task-mytaskapp│mytaskapp
+║my-composed-task-timestamp│timestamp
+
+...
+dataflow:>task destroy my-composed-task
+dataflow:>task list
+╔══════════════════════════╤═══════════════════════════════════════════════════════════════
+║        Task Name         │                      Task Definition
+╠══════════════════════════╪═══════════════════════════════════════════════════════════════
+╚══════════════════════════╧═══════════════════════════════════════════════════════════════
+```
+
+=== Composed Task DSL
+
+==== Conditional Execution
+Conditional execution is expressed using a double ampersand symbol &&.
+This allows each task in the sequence to be launched only if the previous task
+successfully completed. For example:
+```
+task create my-composed-task --definition "foo && bar"
+```
+.Conditional Execution
+image::{dataflow-asciidoc}/images/dataflow-ctr-conditional-execution.png[Composed Task Conditional Execution, scaledwidth="50%"]
+
+When the composed task my-composed-task is launched, it will launch the
+task `foo` and if it completes successfully, then the task `bar` will be
+launched. If the `foo` task fails, then the task `bar` will not launch.
+
+==== Transitional Execution
+The DSL supports fine grained control over the transitions taken during the
+execution of the directed graph. Transitions are specified by providing a
+condition for equality based on the exit status of the previous task.
+A task transition is represented by the following symbol `->`.
+
+===== Basic Transition
+A basic transition would look like the following:
+
+```
+task create my-transition-composed-task --definition "foo 'FAILED' -> bar 'COMPLETED' -> baz"
+```
+
+.Basic Transition
```

Comment:
It seems like UI specific explanations are missing in general. For example, it'd be nice also to include that the transitions in the UI are rendered as dotted lines.